src/lib/components/collections/req-res-section/sub-components/response-body-section/DefaultPage.svelte

```diff
@@ -32,83 +33,50 @@
-        >
-          ctrl + N
-        </button>
-      </div>
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center text-left justify-content-between gap-5 mb-2 mt-2"}
+        key={"Send Request"}
+        value={"ctrl + Enter"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Save Request"}
+        value={"ctrl + S"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"New Request"}
+        value={"ctrl + N"}
+        type="combo"
+      />
     </div>
     <div class={isHorizontalMode ? "d-flex flex-column" : "d-flex flex-column"}>
-      <div class="d-flex align-items-center justify-content-between gap-5 mb-2">
-        <p class="mb-0 shortcut-text">Edit link</p>
-        <button
-          disabled
-          class="bg-buttonBackColor rounded shortcut-btn-api text-buttonColor border-0 py-1 px-4"
-        >
-          alt + L
-        </button>
-      </div>
-      <div class="d-flex align-items-center justify-content-between gap-5 mb-2">
-        <p class="mb-0 shortcut-text">Add Parameter</p>
-        <button
-          disabled
-          class="bg-buttonBackColor rounded shortcut-btn-api text-buttonColor border-0 py-1 px-4"
-        >
-          alt + P
-        </button>
-      </div>
-      <div class="d-flex align-items-center justify-content-center gap-5 mb-2">
-        <p class="mb-0 shortcut-text">Add Header</p>
-        <button
-          disabled
-          class="bg-buttonBackColor rounded shortcut-btn-api text-buttonColor border-0 py-1 px-4"
-        >
-          alt + H
-        </button>
-      </div>
-      <div class="d-flex align-items-center justify-content-center gap-5 mb-2">
-        <p class="mb-0 shortcut-text">Edit Body</p>
-        <button
-          disabled
-          class="bg-buttonBackColor rounded shortcut-btn-api text-buttonColor border-0 py-1 px-4"
-        >
-          alt + B
-        </button>
-      </div>
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Edit link"}
+        value={"alt + L"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Add Parameter"}
+        value={"alt + P"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Add Header"}
+        value={"alt + H"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Edit Body"}
+        value={"alt + B"}
+        type="combo"
+      />

```

Comment:
create an object with keys comboContainerClassProp,key,value,type with their respective values and use map to iterate over the object and pass the values to combotext component

Reply:
resolved